akka-docs/src/main/paradox/typed/interaction-patterns.md

```diff
@@ -368,11 +368,11 @@ which may in worst case cause undesired load on the system. `scheduleWithFixedDe
 
 ## Responding to a sharded actor
 
-The normal pattern for expecting a reply is to include an @apidoc[akka.actor.typed.ActorRef] in the message, typically a message adapter. This can be used
-for a sharded actor but if @scala[`ctx.self`]@java[`ctx.getSelf()`] is sent and the sharded actor is moved or passivated then the reply
-will sent to dead letters.
+When @ref:[Akka Cluster](cluster.md) is used to @ref:[shard actors](cluster-sharding.md) it is recommended to use
```

Comment:
I wouldn't say that it's the (only) recommended approach. I still think ordinary reply via ActorRef is easier in many cases, and message delivery is anyway at-most-once so shouldn't be a big problem for the rare occasions when it would be dropped (and that can happen also when sending via sharding).

The original phrasing was better, but would be good with link and intro sentence about Cluster and sharding.